src/module.ts

```diff
@@ -99,11 +99,13 @@ export default defineNuxtModule<ModuleOptions>({
     await Promise.all([
       addComponent({
         name: 'TresCanvas',
-        filePath: resolver.resolve('./runtime/TresCanvas.client.vue'),
+        filePath: resolver.resolve('./runtime/components/TresCanvas.client.vue'),
+        mode: 'client',
```

Comment:
Hey @danielroe please correct me if I'm wrong, mode is not necessary here right? Nuxt detects it already with the suffix of the component?

Reply:
Just tested again, works with and without file extension and doesn't require the mode field. Removed them both.